src/i18n/locales/en.json

```diff
@@ -88,6 +88,13 @@
       "textarea": "TextArea",
       "clear-all": "Clear All"
     },
+    "slider": {
```

Comment:
please update to "slider-page", otherwise Dashboard won't show Title and Description